cloudinary-core/src/main/java/com/cloudinary/Configuration.java

```diff
@@ -47,6 +47,7 @@
     public boolean forceVersion = true;
     public boolean longUrlSignature = DEFAULT_IS_LONG_SIGNATURE;
     public SignatureAlgorithm signatureAlgorithm = DEFAULT_SIGNATURE_ALGORITHM;
+    public String oauthToken = null;
```

Comment:
maybe call it oAuthToken ?

Reply:
The camelCase name defined in the ticket is`oauth_token` so it kinda translates to oauthToken - but honestly one letter word in variables are always an issue. Not sure what makes more sense. @asisayag2 want to make the call?

Reply:
Looked at other Java OAuth libraries... 
`OAuthBlaBla` for objects
`oauthBlaBla` for members
So `oauthToken` it is.